README.md

```diff
@@ -1,2 +1,18 @@
 # flytectl
 Flyte CLI
+
+In order to dev, needing to work with a cluster.
```

Comment:
Can we move this into a docs/contributing.md file maybe?
I think the main README.md should give a high level view on what this repo is about... maybe installation and building instructions...

Reply:
I'll start a docs/contributing.md with this info.  Seems a good location to also add additional things like building/etc.